### DISCUSSION TITLE
MMKV鸿蒙工程支持

### DISCUSSION THREAD

**Discussion by hezihaog** [Tencent/MMKV#758] [Ideas]
想问下，有计划让MMKV支持鸿蒙工程吗？想在鸿蒙工程上使用MMKV

**Comment by lingol** [comment]
As long as I'm aware, developing for HarmonyOS is not that different from the Android platform.
I believe you could still use Gradle and stuff, no?

**Comment by hezihaog** [comment]
JCenter或MavenCenter中的版本，是安卓使用的aar，而鸿蒙工程不能使用aar，需要使用har。har需要在一个鸿蒙模块中打包来生成。我在Gitee中找到了一个[移植版本](https://gitee.com/hihopeorg/MMKV)，但毕竟不是官方的，后续的BUG修复和更新也不一定及时，所以希望官方能支持上，感谢回复~

**Comment by lingol** [comment]
It's live on OHPM now. #1261
